controllers/apply/form-router-next/lib/application-expiry.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+const path = require('path');
+const get = require('lodash/get');
+const moment = require('moment');
+const config = require('config');
+const enableExpiration = config.get('awardsForAll.enableExpiration');
+
+const { sendHtmlEmail } = require('../../../../common/mail');
+const {
+    PendingApplication,
+    ApplicationEmailQueue
+} = require('../../../../db/models');
+
+const { getEmailFor, getPhoneFor } = require('../../../../common/contacts');
+
+const {
+    signTokenUnsubscribeApplicationEmails
+} = require('../../../user/lib/jwt');
+const { getAbsoluteUrl } = require('../../../../common/urls');
+
+const appData = require('../../../../common/appData');
+const logger = require('../../../../common/logger').child({
+    service: 'application-expiry'
+});
+
+/**
+ * Email Queue Handler:
+ * Checks whether environment variable exists
+ * Calculates time-to-finish-application value
+ * Iterate through emailQueue, within which it:
+ * -- Sends email
+ * -- Updates db
+ * returns an array of objects containing emailSent, dbUpdated for each queue
+ */
+const sendExpiryEmails = async (req, emailQueue, locale) => {
+    logger.info('Handling email queue');
+
+    if (appData.isNotProduction && !process.env.APPLICATION_EXPIRY_EMAIL) {
+        throw new Error(
+            'Missing environment variable APPLICATION_EXPIRY_EMAIL'
+        );
+    }
+
+    return await Promise.all(
+        emailQueue.map(async emailToSend => {
+            let subjectLine = '';
+
+            switch (emailToSend.emailType) {
```

Comment:
💭 Wondering if the `emailQueue` should also contain the subject line / email content? I think this is fine for now though.